mantidproject/mantid
# Fix inconsistent scaling of errors on fit curves by reduced chi-squared

**Description of work.**
To see what breaks for now...
See issue for details

**To test:**
(1) Run the script in the original issue you should see the plot on the RHS (i.e. the error on the fit curve should agree with the 68.3% confidence bounds calculated from the covariance matrix from scipy `curve_fit `with `absolute_sigma=True`
![image](https://user-images.githubusercontent.com/55979119/149945630-1c5f780b-72a4-4bdd-b7b5-33c3f5e82c53.png)


Fixes #33340 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
